cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -3517,13 +3540,14 @@ class RaysView extends PolyShapeView {
         this._lines = [];
         this._z_order = position.z_order;
         const lines = this._splitIntoLines(position.points);
-        lines.forEach(points => {
+        lines.forEach((points, index) => {
             const line = this._uis.shape.polyline(points);
             this._lines.push(line);
             line.fill('inherit').stroke('inherit').attr({
```

Comment:
isn't `inherit` the default? why do you need to specify attributes explicitly?

Reply:
I don't know what default value is, [mozilla docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-width) say it's `1px`.
I wanted to explicitly state that the values come from the parent `group` element.